scss/donate.scss

```diff
@@ -0,0 +1,338 @@
+/*@use 'vars';*/
+//temporary until resolve _vars.scss issue
+
+HERO SECTION BEGINS HERE
+.hero-clouds {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  z-index: 1;
+  @media only screen and (max-width: 767px) {
+    width: 834px;
+    height: 663px;
+    left: -100px;
+    top: -26px;
+  }
+
+  @media only screen and (max-width: 1024px) {
+    width: 834px;
+    height: 663px;
+    left: 0px;
+    top: 69px;
+  }
+}
+
+.hero {
+  position: relative;
+  overflow: hidden;
+  padding-bottom: 10%;
+}
+
+@keyframes wait {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+    pointer-events: auto;
+  }
+}
+
+@mixin fade-in($waitTime) {
+  animation: wait $waitTime, fade-in 800ms $waitTime;
+  animation-duration: 2s;
+}
+
+.hero-information {
+  position: relative;
+  z-index: 3;
+  @include fade-in(600ms);
+  .hero-logo {
+    margin-bottom: 15px;
+  }
+
+  @media only screen and (max-width: 767px) {
+    margin-top: -225px;
+  }
+  #technica-logo {
+    padding-top: 5%;
+    margin-top: 10%;
+    width: 600px;
+    @media only screen and (max-width: 767px) {
+      padding-top: 10%;
+    }
+  }
+  #hero-tagline {
+    font-size: 150%;
+    width: 55%;
+    @media only screen and (max-width: 767px) {
+      font-size: 100%;
+      width: 95%;
+    }
+    @media only screen and (min-width: 767px) and (max-width: 1042px) {
+      width: 85%;
+    }
+  }
+}
+
+.hero-action {
+  position: relative;
+  z-index: 2;
+  opacity: 0;
+  animation: fade-in 2s;
+  animation-delay: 1.5s;
+  animation-fill-mode: forwards;
+
+  @media only screen and (min-width: 996px) {
+    padding-top: 115px;
+  }
+
+  @media only screen and (min-width: 1400px) {
+    padding-top: 350px;
+  }
+
+  .register-btn {
+    @include dinpro-text(175%, bold, #fff);
+    position: relative;
+    z-index: 2;
+
+    @media only screen and (max-width: 767px) {
+      @include dinpro-text(125%, bold, #fff);
+    }
+  }
+  .hero-learn {
+    color: #b6a1c4;
+    font-size: 150%;
+    @media only screen and (max-width: 767px) {
+      font-size: 115%;
+    }
+  }
+}
+
+@media only screen and (max-width: 991px) {
+  .hero-information {
+    .hero-logo {
+      margin-top: 150px;
+    }
+  }
+  .hero-action {
+    padding-top: 100px;
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .hero-information {
+    .hero-logo {
+      margin-top: 225px;
+    }
+    #technica-logo {
+      width: 500px;
+    }
+  }
+  .hero-action {
+    padding-top: 100px;
+  }
+}
+
+@media only screen and (max-width: 575px) {
+  .hero {
+    padding-bottom: 75px;
+  }
+  .hero-information {
+    font-size: 5vw;
+    #technica-logo {
+      width: 80vw;
+    }
+    #theme-logo {
+      width: 40vw;
+      min-width: 120px;
+    }
+  }
+}
+
+/*HERO SECTION ENDS*/
+
+$cyan: #82ecec;
```

Comment:
sorry i didn't check for this before.. but let's delete this & $hero-blue and replace with $blue: $6fcbdc, then replace vars used below